app/models/event.rb

```diff
@@ -14,4 +14,8 @@ def rated_by?(user)
   def average_rating
     ratings.pluck(:rating_score).inject(0, :+) / (ratings.count.nonzero? || 1)
   end
+
+  def rating_for(user)
+    ratings.find_by(user_id: user.id)
+  end
```

Comment:
So how does calling `event.rating_for(user)` know which event ratings we are looking for? Should there not be something like `Rating.find_by(user_id: user.id, event_id: self.id)`? Or is it because the `event` `has_many :ratings`, that you can just go ahead and call `ratings` and it knows that you're referring to this event's ratings?